core/src/avm2/class.rs

```diff
@@ -1155,24 +1156,39 @@ impl<'gc> Class<'gc> {
             &'static str,
             Option<NativeMethodImpl>,
             Option<NativeMethodImpl>,
+            Option<Gc<'gc, Multiname<'gc>>>,
```

Comment:
Wouldn't it be simpler if this accepted `Option<&' static str>` instead?

Reply:
I think the simplest thing here would be to use the `CommonNamespaces` fields in the getter/setter definitions.